canonical-web-and-design/microk8s.io
# Updated global nav to the site

## Done
Changed the installation of the global nav.

## QA
- Open the demo
- See the global nav appears and is the same width as the navigaiton
- Check there are no errors in the console
- Check small screens
